resources/wikia/modules/krux.js

```diff
@@ -0,0 +1,64 @@
+/*global Krux,define*/
+window.Krux || ((Krux = function () {
+	Krux.q.push(arguments);
+}).q = []);
+
+define('wikia.krux', ['wikia.window'], function (win) {
+	'use strict';
+
+	var maxNumberOfKruxSegments = 27;
+
+	function load(confid) {
```

Comment:
@rogatty has a good point here! We don't want to put Krux script every time we call `load()` and we do it every time we change article in Mercury. My idea is to split this method into two: `loadScript()` and `exportPageParams()`. Mercury will call `krux.loadScript()` only with the first page load and then after changing article it'll just call `krux.exportPageParams()`. What are your thoughts @dianafa and @gabrys?